aptos-move/block-executor/src/proptest_types/types.rs

```diff
@@ -467,6 +467,17 @@ where
             Transaction::Abort => ExecutionStatus::Abort(txn_idx as usize),
         }
     }
+
+    fn execute_transaction_in_sequential_execution(
+        &self,
+        view: &impl TStateView<Key = K>,
+        txn: &Self::Txn,
+        txn_idx: TxnIndex,
+    ) -> ExecutionStatus<Self::Output, Self::Error> {
+        // As these functions are dummy, we reuse the code by calling
```

Comment:
So this is just to satisfy the trait? If it is never called maybe having `unreachable!()` is better?

Reply:
This is in proptest folder. So, maybe this is used for testing.

Reply:
Can we know for sure how it is used? Having a good testing infra is important.

If it is used for testing it the comment is misleading then, it is not dummy. Otherwise you can call sequential and compare to parallel, assuming it tests your code but it does not.

If it is not used for testing (explicitly called) then having unreachable there is nicer in my opinion because it will panic in case it is used incorrectly?

Reply:
Or I can just copy-paste the same code the for both _sequential and _parallel functions?

Reply:
Why copy-paste? It is even worse than using a call to `..._parallel`. Why not `unreachable!("Some message")`?